.prettierrc

```diff
@@ -1,6 +1,5 @@
 {
   "singleQuote": true,
   "trailingComma": "es5",
-  "tabWidth": 2,
-  "plugins": ["./node_modules/prettier-plugin-jsdoc/dist/index.js"]
+  "tabWidth": 2
```

Comment:
Can you revert this? At least on git pre-commit hook it formats jsdocs correctly. Later I can help you fixing why our editor isn't formatting.

Reply:
Does removing it breaks husky?